lib/matplotlib/artist.py

```diff
@@ -236,15 +235,16 @@ def stale(self):
 
     @stale.setter
     def stale(self, val):
-        # only trigger call-back stack on being marked as 'stale'
-        # when not already stale
-        # the draw process will take care of propagating the cleaning
-        # process
-        if not (self._stale == val):
-            self._stale = val
-            # only trigger propagation if marking as stale
-            if self._stale:
-                self.pchanged()
+        self._stale = val
+
+        # if the artist is animated it does not take normal part in the
+        # draw stack and is not expected to be drawn as part of the normal
+        # draw loop (when not saving) so do not propagate this change
+        if self.get_animated():
+            return
```

Comment:
Is it still going to be possible to change the properties of an animated object? I typically do things like changing the alpha or color as part of an animation.